### ISSUE TITLE
[Docs] Extremely slow load times

### ISSUE THREAD

**Issue by flexchar** [jgthms/bulma#2008]
This is about **the Docs**.

### Overview of the problem

My **browser** is: Chrome Beta, the type, however, does not influence the issue.

### Description

The website is performing **too slow** and is not acceptable, considering it's staticly based. And thus on average take up to 6 seconds to load a page. It is insanely annyoing when looking for something through docs.

[Testmysite.io](https://testmysite.io/5b51af16fdd72a675c249f0f/bulma.io) shows **load speeds up to 17sec**
https://i.imgur.com/H17EEFB.jpg. 

At the same time, I would strongly suggest moving over to Netlify hosting platform. (From personal experience)

**Comment by jgthms** [comment]
Testmysite.io is run by Netlify, so I'm not sure how unbiased they are tbh.

I did a quick check with [Byte Check](http://www.bytecheck.com/results?resource=bulma.io):
![image](https://user-images.githubusercontent.com/1254808/43016796-e601d47c-8c4b-11e8-8637-61a321b6dbbb.png)

And on [Sucuri](https://performance.sucuri.net/domain/bulma.io):
![image](https://user-images.githubusercontent.com/1254808/43016829-08aed7c2-8c4c-11e8-8693-20333e4517ee.png)

Yeah, I wouldn't trust Testmysite.io that much.

The website is hosted on GitHub because it's really convenient for people to fix documentation issues. And everything it runs Jekyll as well.

I'm not sure Netlify would be a great improvement and I don't want to maintain that either.

**Comment by orliesaurus** [comment]
I have *no* issues with the loading speeds either, from Texas.

**Comment by flexchar** [comment]
Thanks for the prompt reply and sorry for my late reply. 

It now seems fine. 

Since it hosted here, it is very strange as GitHub's CDN is known to even outperform some paid services out there, although I've encountered some slowness experience after recent changes...

Regarding testmysite.io, that is a good question. The root cause of me opening the issue was not those tests but rather my own experience. I used that as an additional information. If I encounter the same again, will post Sucuri tests instead. 

From the maintenance perspective, Netlify perhaps would be even better than GitHub. It supports CD. So basically besides updating DNS and linking Git repo nothing needs to be done.
